pages/dkp/kommander/2.2/install/air-gapped/catalog/host-on-gitea/index.md

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
I had thought about this a bit more and I think we can safely remove this.

For the use case that i was talking about (regarding a non default gitrepo) - we can still use the config to create "A GitRepository", it wont just be default for workspace/project. 

The user can manually copy the GitRepository from one namespace to a namespace of their choice. We can worry about that if/when there is an ask for such a use case. Lets get rid of this page entirely for now. cc @cbuto @gracedo

Reply:
that sounds good to me, I'll remove it!

Reply:
removed in https://github.com/mesosphere/dcos-docs-site/pull/4193/commits/4d5378599eedb1f0c76c79b160ffb73326b65d29